src/components/common/account/info/InfoButton.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import theme from '../../../../theme'
+import { Button, Link } from '@mui/material'
+
+interface Props {
+    buttonUrl: string
+    buttonText: string
+}
+
+export const InfoButton = (props: Props) => {
+    const { buttonUrl, buttonText } = props
+    return (
+        <>
+            <Button
+                sx={{
+                    textTransform: 'none',
+                    height: '39px',
+                    textAlign: 'center',
+                    backgroundColor: theme.palette.primary.main,
+                    borderRadius: 0,
+                    margin: '5px 25px',
+                }}
+            >
+                <Link
+                    href={buttonUrl}
```

Comment:
mam tylko pytanie do czego przekierowuje ten link? Bo jeśli do strony w obrębie apalikacji to chyba lepiej użyć "to" zamiast 'href' Nie wczytywałem się w dokumnetację MU bo może href tam działa tak samo jak 'to' z react router

Reply:
@rafal-szczepanik z jakiegoś powodu Link "to" wyrzuca mi błąd, a href nie.

Reply:
No to jeśli się strona nie odświeża w obrębie aplikacji to href jest ok